protobuf/src/main/java/com/linecorp/armeria/server/protobuf/ProtobufResponseConverterFunction.java

```diff
@@ -71,7 +72,7 @@
 public final class ProtobufResponseConverterFunction implements ResponseConverterFunction {
 
     static final MediaType X_PROTOBUF = MediaType.create("application", "x-protobuf");
-    private static final Printer defaultJsonPrinter = JsonFormat.printer();
+    private static final Printer defaultJsonPrinter = JsonFormat.printer().omittingInsignificantWhitespace();
```

Comment:
This seems like it could be a too big behavior change for a minor version, I guess docs pointing sequence users to pass in a printer is ok?

Reply:
Most users use a JSON parser library instead of manual parsing, so I thought this is not a big change. 😅
Reverted and added a TODO for this.